src/components/form/Form.jsx

```diff
@@ -6,9 +6,13 @@ function withWrapper(Component) {
   return class extends React.Component { // eslint-disable-line react/prefer-stateless-function
     render() {
       const { label, ...props } = this.props; // eslint-disable-line react/prop-types
+
+      const required = (props.validations || []).find(e => e == 'required');
```

Comment:
You're absolutely right. It got lost in the middle of the other eslint complaints. :/ Can we ship #35 and add a precommit hook so this doesn't happen again?